src/ir/var.rs

```diff
@@ -130,6 +130,53 @@ fn default_macro_constant_type(value: i64) -> IntKind {
     }
 }
 
+/// Determines whether a set of tokens from a CXCursor_MacroDefinition
+/// represent a function-like macro. If so, calls the func_macro callback
+/// and returns `Err(ParseError::Continue)` to signal to skip further
+/// processing.  If conversion to UTF-8 fails (it is performed only where it
+/// should be infallible), then `Err(ParseError::Continue)` is returned as well.
+fn handle_function_macro(
+    tokens: &[clang::ClangToken],
+    callbacks: &dyn crate::callbacks::ParseCallbacks,
+) -> Result<(), ParseError> {
+    let mut spelled = tokens.into_iter().map(clang::ClangToken::spelling);
```

Comment:
nit: this can be just `iter()`.

Reply:
I have changed it as you requested. I have a superstitious habit of preferring `.into_iter()` when data is moving anyway, but I should change that habit.